### PR TITLE
tools: fix issues identified by checkers

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: tools/extra/mmlink/mmlink_connection.cpp [hunk 1]
```diff
@@ -44,7 +44,10 @@
 using namespace std;
 
 const char *mmlink_connection::UNKNOWN = "UNKNOWN\n";
+#define MMLINK_UNKNOWN_SIZE  9
```
Comment (r-rojo) [inline tools/extra/mmlink/mmlink_connection.cpp]: Why is this 9?
UNKNOWN\n is eight chars.

File: tools/extra/mmlink/mmlink_connection.cpp [hunk 2]
```diff
@@ -44,7 +44,10 @@
...
+
 const char *mmlink_connection::OK = "OK\n";
+#define MMLINK_OK_SIZE  4
```
Comment (r-rojo) [inline tools/extra/mmlink/mmlink_connection.cpp]: OK\n is three chars

File: tools/extra/mmlink/mmlink_connection.cpp [hunk 3]
```diff
@@ -198,7 +201,7 @@ int mmlink_connection::handle_unbound_command(char *cmd)
 				" setting to bound state\n";
 
 		bind();
-		send(OK, strnlen_s(OK, sizeof(OK)));
+		send(OK, strnlen_s(OK, MMLINK_OK_SIZE));
```
Comment (r-rojo) [inline tools/extra/mmlink/mmlink_connection.cpp]: Seeing as this is C++ already, can't you use C++ strings instead?

Comment (anandaravuri) [reply tools/extra/mmlink/mmlink_connection.cpp]: extra size for "\0" , added +1 to it

**Pull request merged**